packages/@react-spectrum/provider/stories/Provider.stories.tsx

```diff
@@ -85,7 +86,110 @@ storiesOf('Provider', module)
   .add(
     'custom theme',
     () => render({theme: THEME})
-  );
+  )
+  .add(
+    'responsive styleProps',
+    () => (
+      <Provider UNSAFE_style={{padding: 50}}>
+        <div>
+          <TextField
+            label="A text field"
+            placeholder="Something"
+            width={{base: 'size-800', S: 'size-1000', M: 'size-2000', L: 'size-3000'}} />
```

Comment:
What is the expectation here if this was set to something like `width={{base: 'size-3000', M: 'size-2000', S: 'size-1000'}}`? Currently I'm seeing the Textfield width set to `size-2000` at the Large break point, but I thought it would adopt the `base` size of `size-3000`

Reply:
Thanks for checking.

> EDIT: perhaps this is where the user would specify mobileFirst?

Yes, if `mobileFirst` is true and the style prop on `L` has not provided, 
the second biggest breakpoint value will be adopted. (Currently, `mobileFirst` is true as default.)
In this case, `M` (size-2000) is expected.